src/main/java/ContainersMain.java

```diff
@@ -0,0 +1,116 @@
+import java.util.*;
+
+public class ContainersMain {
+
+    public static void main(String[] args) {

```

Comment:
Ai rezolvat problema in main?
De ce?
Cum ai testat acel main?
Daca ai fi avut nevoie de solutia asta intr-o aplicatie web cum ai fi facut?

Nu mai rezolva in main ci fa clase separate si testeaza-le.